admin/views/view_my_items.php

```diff
@@ -215,5 +232,25 @@ function loadMore(size, dataLength, item) {
 			}
 		});
 	}
+
+	window.opinoinstageHandleSelectLinks = function(elm){
+		var url = $(elm).children("option:selected").data('href');
+		if( url ) {
+			window.location = url;
+		}
+	}
+
+	if( $('.opinoinstage-show-anchor-list').length > 0 ) {
+		$(document).click(function(e){
+			if( $(e.target).is('.opinoinstage-show-anchor-list') ) {
```

Comment:
it should be:
```javascript
jQuery(function($) {
  $('.opinoinstage-show-anchor-list').on('click', function (event) {
    event.preventDefault()
   $(this).toggleClass('opened')
  })
})
```
(not sure about exact function names, just describing general idea)

Reply:
if I do it this way - it will not close dropdown if user clicks outside of dropdown